### PR TITLE
add markers to google maps + API keys

### PR DESCRIPTION
**Pull request by mysticeti**

### CODE REVIEW

File: lib/google_maps/google_maps_page.dart [hunk 1]
```diff
@@ -27,6 +32,29 @@ class _GoogleMapsPageState extends State<GoogleMapsPage> {
     controller.animateCamera(CameraUpdate.newCameraPosition(_kPool));
   }
 
+  void _addMarkers(LatLng markerPosition) {
+    final int markerCount = markers.length;
+
+    // prevent excessive markers
+    if (markerCount == 12) {
+      return;
+    }
```
Comment (joykuotw) [inline lib/google_maps/google_maps_page.dart:41]: why have to prevent this?

File: lib/google_maps/google_maps_page.dart [hunk 2]
```diff
@@ -38,9 +66,15 @@ class _GoogleMapsPageState extends State<GoogleMapsPage> {
...
         mapType: MapType.hybrid,
+        myLocationButtonEnabled: false,
+        myLocationEnabled: false,
         initialCameraPosition: _kQueenElizabethOlympicPark,
+        markers: Set<Marker>.of(markers.values),
         onMapCreated: (GoogleMapController controller) {
           _controller.complete(controller);
+          // two different ways to pass variables to _addMarkers
+          _addMarkers(LatLng(51.54265, -0.00956));
+          _addMarkers(kPoolLocation);
```
Comment (joykuotw) [inline lib/google_maps/google_maps_page.dart:77]: No need to give a comment as this is quite obvious.

**Review by joykuotw — CHANGES_REQUESTED** [review]

File: lib/google_maps/google_maps_page.dart [hunk 3]
```diff
@@ -27,6 +32,29 @@ class _GoogleMapsPageState extends State<GoogleMapsPage> {
...
+      markerId: markerId,
+      position: markerPosition,
+      infoWindow: InfoWindow(title: markerIdVal, snippet: '*'),
+    );
+
+    setState(() {
+      markers[markerId] = marker;
+    });
+  }
+
```
[the remark this answers is not in the document]
Comment (mysticeti) [reply lib/google_maps/google_maps_page.dart:57]: renamed it to _addMarker in the next commit.

File: lib/google_maps/google_maps_page.dart (hunk 1 above)
Comment (mysticeti) [reply lib/google_maps/google_maps_page.dart:41]: I will remove it in the next commit. It was suggested in the google maps example which is why I added it.

File: lib/google_maps/google_maps_page.dart (hunk 2 above)
Comment (mysticeti) [reply lib/google_maps/google_maps_page.dart:77]: Removing the comment.

**Review by joykuotw — APPROVED** [review]

**Pull request merged**